automation/run-tests.sh

```diff
@@ -97,7 +97,7 @@ function run_tests {
             echo "Running unit test in $DOCKER_IMAGE container is not " \
                  "support yet"
         else
-            container_exec 'tox -e check-py36'
+            container_exec 'tox -e py36'
```

Comment:
I find this name change strange.
`check` points to the run of unit tests, but now it has a generic name of `pyXY` which says nothing compared to the other target names.
If tox interprets the python version from the target name, then isn't it able to do the same with some descriptive prefix as well?